### ISSUE TITLE
Correct probabilities when doing weighted walk

### ISSUE THREAD

**Issue by alongalky** [iotaledger/iotavisualization#30]

**Comment by mlouielu** [comment]
Yep, it seems unweighted walk is better than weighted walk in current version (orange tx is the cumulative weight greater than the filter value).


unweighted walk:
![](https://i.imgur.com/Z6imtAk.png)

weighted walk:
![](https://i.imgur.com/ks2PIKh.png)

**Comment by alongalky** [comment]
Whether it's "better" or not depends what we want to achieve. You're using a very high value of alpha, which causes a "narrow corridor effect", with a lot of transactions left behind (notice there are many tips as well).

Just to clarify: the calculation is correct, this issue is only referring to displaying the animation correctly (only relevant for the one-by-one branch until that is merged in to master)

**Comment by alongalky** [comment]
BTW, this is a prerequisite to merging the one-by-one branch
